### ISSUE TITLE
publish-TR-webgpu: Attribute “xlink” not allowed on element “svg” at this point

### ISSUE THREAD

**Issue by kainino0x** [gpuweb/gpuweb#3222]
The publish-TR-webgpu step seems to have broken a few days ago https://github.com/gpuweb/gpuweb/actions/workflows/publish-TR-webgpu.yml

I'm not sure what's up with the error, haven't had a chance to investigate. Unsure if it affects all PRs.

```
Validate markup
  Validating index.html...
  $ yarn add vnu-jar --no-progress
  └─ vnu-jar@21.10.12
  $ java -jar "/home/runner/work/_actions/w3c/spec-prod/v2/node_modules/vnu-jar/build/dist/vnu.jar" --also-check-css index.html
    "file:/home/runner/work/gpuweb/gpuweb.w3c/spec/index.html":2224.6-2224.158: error: Attribute “xlink” not allowed on element “svg” at this point.
  ❌  Not so good... please fix the issues above.
```

cc @tidoust

**Comment by kainino0x** [comment]
Thanks @tidoust for debugging and filing the Bikeshed issue!

This reminds me, is there a way to get spec-prod to use a pinned version of Bikeshed? It would be convenient to avoid build breakage, plus we're already pinning a version of Bikeshed for some of the other build steps so it would be more consistent.

**Comment by tidoust** [comment]
> This reminds me, is there a way to get spec-prod to use a pinned version of Bikeshed? It would be convenient to avoid build breakage, plus we're already pinning a version of Bikeshed for some of the other build steps so it would be more consistent.

There is no way to pin a specific version in spec-prod. That could perhaps be added. I filed https://github.com/w3c/spec-prod/issues/141.

I note disabling HTML validation is not going to work for publication to /TR because the publication rules checker will refuse to publish a document with invalid HTML content in any case.

**Comment by kainino0x** [comment]
> I note disabling HTML validation is not going to work for publication to /TR because the publication rules checker will refuse to publish a document with invalid HTML content in any case.

Thanks for the heads up. Disabling the HTML validation has at least unblocked us from landing commits for now.

**Comment by kainino0x** [comment]
This has been fixed in Bikeshed 3.11.5+, but upgrading causes some new warnings. Filed https://github.com/speced/bikeshed/issues/2471

**Comment by toji** [comment]
Linked Bikeshed issue indicates the problem causing the warnings was fixed. What, if anything, needs to be done on our end still?

**Comment by kainino0x** [comment]
#3229 was already reverted as part of #3265, and you fixed that in #4580. So should be all good!
